common/v2/features/DeFiZap/components/ZapIndicators.tsx

```diff
@@ -12,15 +12,17 @@ interface SProps {
   color: string;
 }
 
-const SContainer = styled.div`
+const SContainer = styled.div<{ txtPosition?: string }>`
   display: flex;
   justify-content: center;
-  align-items: top;
+  align-items: ${props => (props.txtPosition ? props.txtPosition : 'center')};
+  height: 2em;
 `;
 
 const SImage = styled.img`
   height: 24px;
   width: 24px;
+  margin: 15px 0;
```

Comment:
This appears to have broken the indicators: 

![image](https://user-images.githubusercontent.com/29407814/76567075-42031d00-646b-11ea-92aa-5f32cf400f8c.png)

![image](https://user-images.githubusercontent.com/29407814/76567099-4a5b5800-646b-11ea-849c-0a21d0f4a553.png)

Reply:
Oh I didn't expected it to be used here 😱, going to fix that

Reply:
Oh lol sorry, I thought from the designs that depending of the Indicator the title should be up/down or aligned with the icon 😂